operations/network/customer_access_network/bi-can_arista_metallb_peering.md

```diff
@@ -0,0 +1,138 @@
+# Metallb peering with arista edge router
+
+This is an example configuration of how to connect a pair of Arista switches to metallb running inside of kubernetes.
+
+## Prerequisites
+
+- Pair of arista switches already connected to the high speed network.
+- Updated SLS file that has the `CHN` network configured.
+
```

Comment:
Add a new header here to separate from prereqs. Maybe... `## Example Configuration`